src/github.ts

```diff
@@ -1333,6 +1291,21 @@ export class GitHub {
         pullRequest.labels
       );
 
+      // attempt to enable auto-merge
+      if (options?.autoMerge) {
+        try {
+          await this.enablePullRequestAutoMerge(
+            pullRequestNumber,
+            options.autoMerge.mergeMethod
+          );
+        } catch (e: unknown) {
+          this.logger.warn(
+            isOctokitGraphqlResponseError(e) ? e.errors || [] : (e as {}),
+            'Failed to enable auto merge. Continuing.'
+          );
+        }
+      }
+
       // assign reviewers
       if (options?.reviewers) {
```

Comment:
We probably shouldn't assign reviewers if directly merging the PR succeeds, right?

Reply:
I think you're correct, but it's a bit more tricky to check correctly if it has already been merged or not due to all the different type of branch protection, the auto-merge behaviours, etc.

I can handle here the simple case: if the PR state is "clean", and we merge it directly, then we don't assign.

For other situation we can just not configure any reviewers for the repository, as part of the release-please config file.